Tools/Scripts/compare-static-analysis-results

```diff
@@ -24,22 +24,23 @@
 import os
 import subprocess
 import argparse
-import sys
+import json
 
 CHECKERS = ['NoUncountedMemberChecker', 'RefCntblBaseVirtualDtor', 'UncountedCallArgsChecker', 'UncountedLocalVarsChecker']
 PROJECTS = ['WebCore', 'WebKit']
 
 
 def parser():
     parser = argparse.ArgumentParser(description='compare dirty file lists')
-    parser.add_argument(
-        'archived_dir',
-        help='directory of dirty lists from previous build'
-    )
     parser.add_argument(
         'new_dir',
         help='directory of dirty lists from new build'
     )
+    parser.add_argument(
+        '--archived-dir',
+        dest='archived_dir',
+        help='directory of dirty lists from previous build'
```

Comment:
While we're touching this, can we make it a formal sentance with capitalization and punctuation?